src/models/TalkCommentMapper.php

```diff
@@ -386,4 +386,37 @@ public function moderateReportedComment($decision, $comment_id, $user_id)
             }
         }
     }
+
+    /**
+     * @param integer $comment_id
+     * @return false|array
+     */
+    public function getRawComment($comment_id)
+    {
+        $sql = "select tc.* from talk_comments tc where tc.ID = :comment_id";
+        $stmt = $this->_db->prepare($sql);
+        $response = $stmt->execute(array(':comment_id' => $comment_id));
+        if ($response) {
+            $results = $stmt->fetchAll(PDO::FETCH_ASSOC);
+            if (isset($results[0])) {
+                return $results[0];
+            }
+        }
+
+        return false;
+    }
+
+    /**
+     * @param integer $comment_id
+     * @param string $new_comment_body
+     */
+    public function updateCommentBody($comment_id, $new_comment_body)
+    {
+        $sql = "update talk_comments set comment = :new_comment_body where ID = :comment_id";
+        $stmt = $this->_db->prepare($sql);
+        $stmt->execute([
```

Comment:
Check for the return-value of the execute-command here please. Or return it. THat way the caller of the method gets an information whether the method succeeeded or failed and can react appropriately.